cubic-server/Operator.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef CUBICSERVER_OPERATOR_HPP
```

Comment:
This file is named `Operator.hpp` but the content is a class named Permissions.
Either change the class name or the file name as this is pretty confusing.